### PR TITLE
Please review pull request to add ham sandwich recipe for lunch

### PR DESCRIPTION
**Pull request by cristinap37**
## 🎯 Objective

adding ham sandwich recipe for lunch

## 🛠️ Changes made

added all components of recipe for lunch

## 👥 Who needs to review this Pull Request? 
@IntroToCode/fall20_codemates2 please review!!

### CODE REVIEW

**Review by alyssamzhu — COMMENTED** [review]
@cristinap37 this looks great! i wonder if we can add one more item to beef up the sandwich? perhaps we can add avocado and mixed greens too?

File: lunch/readme.md [hunk 1]
```diff
@@ -1,16 +1,21 @@
...
 
-## Time to prepare: 
+## Time to prepare: 5 minutes
 
-## Ingredients:
+## Ingredients: ham, bread, mayo, lettuce
 
 
-## Equipment needed:
+## Equipment needed: butter knife
```
Comment (lindalee117) [inline lunch/readme.md:10]: Proposal: [add] Cutting board

File: lunch/readme.md [hunk 2]
```diff
@@ -1,16 +1,21 @@
...
+# Ham Sandwich
 
-## Servings: 
+## Servings: 1
 
-## Time to prepare: 
+## Time to prepare: 5 minutes
 
-## Ingredients:
+## Ingredients: ham, bread, mayo, lettuce
```
Comment (lindalee117) [inline lunch/readme.md:7]: Proposal: [add] avocado, mixed greens

**Comment by lindalee117** [comment]
@cristinap37 thanks for this! agree it looks good, but we should also add cutting board to equipment needed. Left the changes for review in the file!

**Review by lindalee117 — CHANGES_REQUESTED** [review]
@cristinap37 thanks for this! agree it looks good, but we should also add cutting board to equipment needed. Left the changes for review in the file!

**Review by lindalee117 — APPROVED** [review]

**Pull request merged**